# HG changeset patch
# User John Donoghue <john.donoghue@ieee.org>
# Date 1426726851 14400
#      Wed Mar 18 21:00:51 2015 -0400
# Node ID 23c0f89700a4b69fb808efcfff77dcd1cd31128b
# Parent  8ba35b7b355d14010b51fe4098cd8acc980f828f
configure.ac: set cross_mkoctfile = no if not cross compiling

* configure.ac: when warning that cant create cross_mkoctfile, set variable to no

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2978,16 +2978,17 @@ cross_mkoctfile=no
 AC_ARG_ENABLE([cross-mkoctfile],
   [AS_HELP_STRING([--enable-cross-mkoctfile],
     [build cross-mkoctfile if cross compiling])],
   [if test "$enableval" = yes; then cross_mkoctfile=yes; fi])
 
 if test "$cross_mkoctfile" = yes; then
   if test "$cross_compiling" = no; then
     AC_MSG_WARN([ignoring --enable-cross-mkoctfile when not cross compiling])
+    cross_mkoctfile=no
   fi
 fi
 AM_CONDITIONAL([AMCOND_CROSS_MKOCTFILE], [test $cross_mkoctfile = yes])
 
 
 ### Decide whether or not to install build logs with Octave.
 
 install_build_logs=no
